modules/ROOT/pages/export-to-exchange-task.adoc

```diff
@@ -3,37 +3,71 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio allows you to connect to Exchange to publish your project as a new asset.
+Anypoint Studio enables you to write documentation in your new projects and to connect to Exchange to publish your project as a new asset.
 
-. Right-click your project folder in your Package Explorer view.
+== Publishing Assets with Documentation
+
+When creating a new project in Anypoint Studio, it includes an exchange-docs folder with a home.md file to write the front page of a new asset in Markdown format after publishing it in Exchange.
+
+NOTE: Learn more about Markdown at https://www.markdownguide.org/.
+
+Studio enables you to add more pages, images, and an icon: 
+
+. Create a new project in Anypoint Studio.
+. In the Package Explorer view, look for *Exchange-docs* > *Home.md* and open it.
+. Use the default Text Editor for Studio, or choose an external editor by right clicking on *Open with...* > *Other*, and selecting an external app.
+. Optionally, create additional pages to add to your asset documentation by right clicking on the *Exchange-docs* folder and selecting *New* > *File*.
++
+NOTE: The name of the file will be the name of the page in Exchange.
++
+. In case of having more than two pages, create a `config.json` file to provide page order following the format:
++
+----
+{"pageOrder": ["home","page1","page2"]}
+----
++
+If no format is provided, the home page will be followed by the other pages in alphabetical order. 
++
+. Assign an icon to your project by adding an image (*.jpg, *.png, *.bmp) with the name icon before publishing your project for the first time.
+. Create a *Resources* folder in the *Exchange-docs* folder to add extra resources to your documentation, such as images, or other types of files (size limit of 2MB).
+. To publish your project, right click your project in the *Package Explorer*, and click on *Anypoint Platform* > *Publish to Exchange*.
+. Log in to the platform if needed, fill in the required fields of the two pages, and click on the *Finish* button. 
+. After uploading the asset, click on the link provided to verify that your asset and documentation are available in Exchange.
+
+== Publishing a Project to Exchange
+
+. Right-click your project folder in your *Package Explorer* view.
```

Comment:
You did not bold this previously - the formatting should be consistent.

Reply:
Good catch! Thanks!